<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="run_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>619</y>
      <width>100</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(85, 170, 127)</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QTableView" name="gearTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>871</width>
      <height>431</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="select_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select recording</string>
    </property>
   </widget>
   <widget class="QLabel" name="gear_table_title">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>47</width>
      <height>13</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Inventory</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_box_title">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>871</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>520</y>
      <width>351</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_box">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>70</y>
      <width>351</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Click row to see Gear Image</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="export_btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>619</y>
      <width>100</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(85, 170, 255)</string>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>560</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>No file selected.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>620</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Clear Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end_task_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>620</y>
      <width>100</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionWeb_version"/>
    <addaction name="actionGit"/>
    <addaction name="actionDonate"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="Webapp">
   <property name="text">
    <string>Buy me a coffee</string>
   </property>
  </action>
  <action name="actionBuy_me_a_coffee">
   <property name="text">
    <string>Webapp</string>
   </property>
  </action>
  <action name="actionWeb_version">
   <property name="text">
    <string>Web version</string>
   </property>
  </action>
  <action name="actionGit">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Buy me a coffee</string>
   </property>
  </action>
  <action name="actionTesseract_OCR">
   <property name="text">
    <string>Tesseract-OCR</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
